internal/pkg/api/server/templates/dependencies/dependencies.go

```diff
@@ -60,6 +66,30 @@ func (v *container) CtxCancelFn() context.CancelFunc {
 	return v.ctxCancelFn
 }
 
+func (v *container) DefaultRepository() (model.TemplateRepository, error) {
+	if strings.HasPrefix(v.repositoryPath, "file://") {
+		return model.TemplateRepository{
+			Type: model.RepositoryTypeDir,
+			Name: repository.DefaultTemplateRepositoryName,
+			Url:  strings.TrimPrefix(v.repositoryPath, "file://"),
```

Comment:
Super, len by som z toho spravil private funkciu `defaultRepository`, a hodnotu cachoval v `container`, takto sa to generuje vzdy znova pri kazdom requeste.

Tj. v `NewContainer` by bolo namiesto `repositoryPath: ...`, -> `defaultRepository: defaultRepository(repoPath)`.
Ak je to teda mozne, a neunikla mi nejaka dolezita vec.

Reply:
Cachovat to můžu, ale volá se to ze `service` package, takže private být nemůže.